cmd/entrypoint-wrapper/main_test.go

```diff
@@ -0,0 +1,119 @@
+package main
+
+import (
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+
+	"github.com/openshift/ci-tools/pkg/api"
+)
+
+func TestManageCLI(t *testing.T) {
+	for _, testCase := range []struct {
+		testName string
+		cliDir   string
+		expected string
+	}{
+		{
+			testName: "If CLI directory is set, append it to the PATH environment variable",
```

Comment:
I would make the test names shorter, something like: CLI directory set
The code is self-explanatory